presto-main/src/main/java/io/prestosql/connector/informationschema/InformationSchemaMetadata.java

```diff
@@ -247,98 +248,54 @@ public ConnectorTableProperties getTableProperties(ConnectorSession session, Con
     {
         InformationSchemaTableHandle table = (InformationSchemaTableHandle) handle;
 
-        if (!table.getPrefixes().equals(defaultPrefixes())) {
+        // The table is catalog-wise and doesn't have a schema/table name filter.
+        if (!table.getSchemaPrefixes().isPresent()) {
             return Optional.empty();
         }
 
-        Set<QualifiedTablePrefix> prefixes = getPrefixes(session, table, constraint);
-
-        table = new InformationSchemaTableHandle(table.getCatalogName(), table.getSchemaName(), table.getTableName(), prefixes);
-
-        return Optional.of(new ConstraintApplicationResult<>(table, constraint.getSummary()));
-    }
-
-    private Set<QualifiedTablePrefix> defaultPrefixes()
-    {
-        return ImmutableSet.of(new QualifiedTablePrefix(catalogName));
-    }
-
-    private Set<QualifiedTablePrefix> getPrefixes(ConnectorSession session, InformationSchemaTableHandle table, Constraint constraint)
-    {
-        InformationSchemaTableHandle handle = checkTableHandle(table);
+        // TODO: constraint may have two levels of filters: schema-level and table-level. It also has two types of filters: discrete (=) and consecutive (>, <, LIKE, etc.).
```

Comment:
s/consecutive/range

remove "LIKE" (it doesn't get pushed unless it can be replaced with `=`)